### PR TITLE
Measure coverage: FeedbackQuestionsLogic::getRecipientsForQuestion (Resolve #2)

### LINKED ISSUE #2: Measure coverage: FeedbackQuestionsLogic::getRecipientsForQuestion

### PR DESCRIPTION
**Pull request by gingemarsson**

### CODE REVIEW

**Comment by jacobhal** [comment]
This all looks good to me. However, according to the instructions the changes related to branch coverage should be on a separate development branch since this code is not permanent. Quote: "Use a separate development branch on git for this, as this code is not permanent." Since we do not want these changes to be included when writing unit tests, this should probably be merged into another branch and not master?

**Comment by gingemarsson** [comment]
Good point! I do not have permission to create such a branch, but as soon as @afTrolle creates one I will change the target of the pull request.

File: src/main/java/teammates/logic/core/CoverageCounter.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * Init the coverage counter
+     * @param newSize The total number of branches
+     */
+    public static void init(int newSize) {
+        size = newSize;
+        for (int i = 0; i < 200; i++) {
+            coverage[i] = false;
+        }
+    }
```
Comment (afTrolle) [inline src/main/java/teammates/logic/core/CoverageCounter.java]: Borde inte coverage  array:en allokeras här med size?
Sen själva boolean primitiven är default värde false så behövs inte loop:as.
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html

File: src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java [hunk 2]
```diff
@@ -339,51 +341,65 @@ private boolean areQuestionNumbersConsistent(List<FeedbackQuestionAttributes> qu
...
             List<TeamDetailsBundle> teams = coursesLogic.getTeamsForCourse(question.courseId);
             for (TeamDetailsBundle team : teams) {
                 // Ensure student('s team) does not evaluate own team.
                 if (!giverTeam.equals(team.name)) {
+                    CoverageCounter.covered(7);
                     // recipientEmail doubles as team name in this case.
                     recipients.put(team.name, team.name);
                 }
+                else {CoverageCounter.covered(8);}
             }
```
Comment (afTrolle) [inline src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java]: Borde inte for loopen själv vara en branch? 
Exempel om TeamDetailsBundle är null eller inte innehåller några  teams själva.
Då har själva getter funktionen körts  och case Teams har hänt men ingen  coverage har hänt.

**Pull request merged**